python/lsst/cp/verify/verifyBias.py

```diff
@@ -68,18 +69,25 @@ def verify(self, exposure, statisticsDict):
         """
         detector = exposure.getDetector()
         ampStats = statisticsDict['AMP']
+        metadataStats = statisticsDict['METADATA']
 
         verifyStats = {}
         success = True
         for ampName, stats in ampStats.items():
             verify = {}
-
+            readNoiseMatch = True
             # DMTN-101 Test 4.2: Mean is 0.0 within noise.
             verify['MEAN'] = bool(np.abs(stats['MEAN']) < stats['NOISE'])
 
             # DMTN-101 Test 4.3: Clipped mean matches readNoise.
-            amp = detector[ampName]
-            verify['NOISE'] = bool(np.abs(stats['NOISE'] - amp.getReadNoise())/amp.getReadNoise() <= 0.05)
+            readNoise = detector[ampName].getReadNoise()
+            if 'RESIDUAL STDEV' in metadataStats and ampName in metadataStats['RESIDUAL STDEV']:
```

Comment:
Should we add a comment here, or above, explaining `RESIDUAL STDEV` more? For example, saying that it is the standard deviation of the overscan region after the overscan has been removed., and that this should be identical to the true read noise (according to an explanation I got from you once). This given that the name doesn't seem to be that descriptive.

Reply:
Explanation added here and in verifyDark.  Minor differences also removed in that check (dark checked that `overscanReadNoise` was found, bias did not).